packages/venia-concept/src/components/Gallery/gallery.js

```diff
@@ -17,7 +17,9 @@ class Gallery extends Component {
             shape({
                 id: number.isRequired,
                 name: string.isRequired,
-                small_image: string.isRequired,
+                small_image: shape({
+                    url: string.isRequired
+                }),
```

Comment:
In this component, the prop type for `data` should be simply `array`. The current `arrayOf(shape())` is unnecessary because `Gallery` does nothing with the `data` prop but send it along to `GalleryItems`; even if `data` were null, this component would not throw during render, even if one of its children might.

Reply:
Made this suggested change